sdk/servicebus/service-bus/rollup.base.config.js

```diff
@@ -74,14 +74,7 @@ export function nodeConfig({ test = false, production = false } = {}) {
     baseConfig.output.file = "test-dist/index.js";
 
     // mark assert as external
-    baseConfig.external.push(
-      "assert",
-      "fs",
-      "path",
-      "@azure/arm-servicebus",
-      "@azure/ms-rest-nodeauth",
-      "@azure/identity"
-    );
+    baseConfig.external.push("assert", "fs", "path", "@azure/arm-servicebus", "@azure/identity");
```

Comment:
`"@azure/arm-servicebus"` is not needed too :)

```suggestion
    baseConfig.external.push("assert", "fs", "path", "@azure/identity");
```